attack_range.py

```diff
@@ -51,10 +51,17 @@ def init(args):
     log = logger.setup_logging(config['log_path'], config['log_level'])
     log.info("INIT - attack_range v" + str(VERSION))
 
+    if config['cloud_provider'] == 'azure':
+        os.environ["AZURE_SUBSCRIPTION_ID"] = config['azure_subscription_id']
+
     if config['attack_range_password'] == 'Pl3ase-k1Ll-me:p':
         log.error('ERROR: please change attack_range_password in attack_range.conf')
         sys.exit(1)
 
+    if config['cloud_provider'] == 'azure' and config['zeek_sensor'] == '1':
+        log.error('ERROR: zeek sensor only available for aws in the moment. Plase change zeek_sensor to 0 and try again.')
```

Comment:
"zeek sensor only available in aws for the moment. Please set zeek_sensor to 0 and try again"

Anyways, what about instead showing a warning and ignore that zeek is set to 1?

Reply:
we are so verbose in logging. Probably you will not recognize and then the user is maybe upset because he waited for 30 min and no zeek box is there.